pagopa/io-app
# fix: [IA-598] Fix wrong name surname value in ZendeskAskPermissions screen

## Short description
This PR fix the wrong name surname value shown in ZendeskAskPermissions screen.
This was due to the use of the wrong selector.

## List of changes proposed in this pull request
- Substituted the `profileNameSelector` with the `profileNameSurnameSelector`

<img src="https://user-images.githubusercontent.com/11773070/147230852-71c4ac88-59a2-4fbd-ae19-c56240677041.png" width=300>

## How to test
Try to open a ticket after the login and verify that in the ZendeskAskPermissions are shown both the name and the surname
